packages/ajna-contracts/scripts/common/config.ts

```diff
@@ -58,7 +58,7 @@ export const TOKENS = {
     RETH: "0x0000000000000000000000000000000000000000",
     WSTETH: "0x0000000000000000000000000000000000000000",
     WBTC: "0x0000000000000000000000000000000000000000",
-    USDC: "0x6e4c6e76b3C1D834c0e3c4c2bAec8d58B8421A99",
+    USDC: "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
```

Comment:
Is this correct?

Reply:
yes: https://basescan.org/token/0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913